discover/reference/articles/04-form-fields.markdown

```diff
@@ -0,0 +1,66 @@
+# Form Field Types [](id=form-field-types)
+
+A form without fields is no form at all. To meet your form-building needs,
+Liferay Forms gives you many useful and highly configurable field types.
+
+![Figure 1: Out of the box form field types.](../images/forms-field-types.png)
+
+Form Text
+: Users do not enter data into Form Text fields. The form creator enters text
+that form users see displayed on the form. This is useful for longer
+instructions.
+
+![Figure 2: A form text field.](../images/forms-form-text.png)
+
+Text Field : Users enter text into these fields. For example, a Full Name field
+will often be a text field. By default, a text field keeps all input on a single
+line of text. To accommodate longer responses, choose the multiline setting when
+configuring the text field. Place limits on the text users can enter (numbers
+from 1-10, or email addresses, for example) by using the Text field's validation
+options.
```

Comment:
I think it'd be better to add a link to the forms section of the user guide to the intro, so the reader understands that the usage of the fields is illustrated there. Let me know what you think of this: https://github.com/rbohl/liferay-docs/commit/f424bb173b0e280d53ae1aed02c6d16d89f99bc0

Reply:
Your suggestion is nice, because it covers all the fields and not only the Text field, although I believe that would be nice to include a shortcut to the reader access the Text field configuration. Therefore, I think the both suggestions can be used, once they complement themselves.

But please feel free to take a decision, once you've the view of the whole documentation scenario. 😉